### ISSUE TITLE
About translate of documentation to russian language

### ISSUE THREAD

**Issue by vivazzi** [awesto/django-shop#476]
Hi @jrief, @rfleschenberg and other devs!

Now I translate documentaion of django-shop to russian language. At this moment I translated 18 pages of documentation and place this translation to my blog http://vivazzi.ru/django-shop/

I'd want to provide support of russian translate. This documentaion is updates othen and it is difficulty for translate to keep actually.

How I can translate docs and know changes. Maybe use versions as django-cms docs?

**Comment by rfleschenberg** [comment]
Maybe we should consider using https://wwwtransifex.com.

**Comment by vivazzi** [comment]
Yes, I thought about it. As I know Transifex is not configured for django-shop yet.

**Comment by vivazzi** [comment]
Translate of Django for russian devs is djbook.ru. Now translate of django-shop is my blog. I think that it will be difficult for the traslate through transifex because django-shop is not separated words and phrases. Django-shop as other docs consists of whole sentenses which it is translated with language methods such as joining of sentences, separate of sentences, replace parts of sentences and others. How do you think about it?

**Comment by jrief** [comment]
@vivazzi it would be great, if you could create a pull request which adds all the Transiflex features. Configuring Transiflex is a PITA, therefore I always try to avoid it, whenever possible.

There are some alternatives out there, if you want to play http://alternativeto.net/software/transifex/

**Comment by vivazzi** [comment]
What is PITA?

**Comment by jrief** [comment]
too vulgar to explain :)

(pain in the ass)

**Comment by vivazzi** [comment]
Oh, Okey :)

Maybe I can configure transiflex, but it will be first for me... Transiflex may help translate django-shop, but django-shop documentaion is cmore complex. Our russian translate command for django (docs.djangoproject.com) is djbook.ru. I think that first of time I keep translate of django-shop docs in my blog. Furthermore, my translate is not good and I'm going to improved its with russian supporters. Now we all need django-shop docs by version (as django-cms http://django-cms.readthedocs.io/). The translation will be more easy with version when the changes will be added to docs.
